sdk/include/game-contract-sdk/dispatcher.hpp

```diff
@@ -83,3 +97,39 @@ bool execute_action(eosio::name self, eosio::name code, void (game::*func)(Args.
         eosio::eosio_exit(0);                                                                                          \
     }                                                                                                                  \
     }
+
+
+#define GAME_CONTRACT_CUSTOM_ACTIONS(TYPE, MEMBERS)                                                                    \
```

Comment:
redefine `GAME_CONTRACT` to avoid copy-paste like: `#define GAME_CONTRACT(TYPE) GAME_CONTRACT_CUSTOM_ACTIONS(TYPE, )`

Reply:
fixed